presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/CheckUnsupportedPrestissimoTypes.java

```diff
@@ -33,20 +33,28 @@
 import com.facebook.presto.spi.relation.RowExpression;
 import com.facebook.presto.spi.relation.SpecialFormExpression;
 import com.facebook.presto.spi.relation.VariableReferenceExpression;
+import com.facebook.presto.sql.analyzer.FeaturesConfig;
 import com.facebook.presto.sql.parser.SqlParser;
 import com.facebook.presto.sql.planner.SimplePlanVisitor;
 import com.facebook.presto.sql.planner.TypeProvider;
 import com.facebook.presto.sql.planner.plan.WindowNode;
 
 import java.util.List;
+import java.util.Objects;
 
 import static com.facebook.presto.common.type.TimestampWithTimeZoneType.TIMESTAMP_WITH_TIME_ZONE;
+import static com.facebook.presto.type.IpAddressType.IPADDRESS;
 import static com.google.common.base.Preconditions.checkState;
 
-public class CheckNoTimestampWithTimezoneType
+public class CheckUnsupportedPrestissimoTypes
         implements PlanChecker.Checker
 {
-    private static final String errorMessage = "Timestamp with Timezone type is not supported in Prestissimo";
+    private static FeaturesConfig config;
```

Comment:
Why have it as a static member?

Reply:
It was due to the classes and functions being in a static context. Removed static from all of them.